src/templates/components/CapiSingleCard.svelte

```diff
@@ -5,7 +5,6 @@
 	import AudioIcon from './icons/AudioIcon.svelte';
 	import CameraIcon from './icons/CameraIcon.svelte';
 	import VideoIcon from './icons/VideoIcon.svelte';
-	import SponsorRedesign from './SponsorRedesign.svelte';
```

Comment:
Im assuming this `SponsorRedesign` file has been replaced by the `Sponsor`? Does that need t be imported here?